docs/plan/huberman-knowledge-pipeline/phase-4-protocol-upgrade/cursor-audit/protocol-data-flow.md

```
@@ -0,0 +1,111 @@
+     ▼                                                                 ▼
+formatHealthPlanForTelegram(healthPlan)                         Telegram (basic plan)
+[schedule-engine.ts]                                                    │
+     │                                                                 │
+     ▼                                                                 │
+Telegram (/schedule command)                                           │
+```
+
+---
+
+## Entry Points Summary
+
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)